tools/wine4office-manager/tests/test_backend.py

```
@@ -1058,6 +1073,165 @@ def test_teams_detection_prefers_newest_windowsapps_package_and_creates_shortcut
+                outside = self.home / ancestor.replace("/", "-") / "outside"
+                windows_apps = outside / "WindowsApps"
+                package = windows_apps / (
+                    "Microsoft.OutlookForWindows_1.2026.900.100_x64__8wekyb3d8bbwe"
+                )
+                package.mkdir(parents=True)
+                (package / "olk.exe").write_bytes(b"outside")
+                symlink = prefix / ancestor
+                if symlink.exists():
+                    symlink.rmdir()
+                symlink.parent.mkdir(parents=True, exist_ok=True)
+                target = outside if ancestor == "drive_c" else outside
```

Ruff (0.16.1)

[warning] Useless `if`-`else` condition

(RUF034)